.github/styles/vocab.txt

```diff
@@ -75,6 +75,7 @@ Firekube
 Fiverr
 freben
 Fredrik
+gitbeaker
 github
 GitHub
```

Comment:
Unrelated: I would assume we always want to write GitHub correctly cased. Maybe I'll have a look to that.